share/availability/cache/testing.go

```diff
@@ -0,0 +1,39 @@
+package cache
+
+import (
+	"testing"
+
+	ds "github.com/ipfs/go-datastore"
+	dssync "github.com/ipfs/go-datastore/sync"
+	mdutils "github.com/ipfs/go-merkledag/test"
+
+	"github.com/celestiaorg/celestia-node/share"
+	"github.com/celestiaorg/celestia-node/share/availability/full"
+	"github.com/celestiaorg/celestia-node/share/availability/light"
+	availability_test "github.com/celestiaorg/celestia-node/share/availability/test"
+	"github.com/celestiaorg/celestia-node/share/service"
+)
+
+// RandLightLocalServiceWithSquare is the same as randLightServiceWithSquare, except
```

Comment:
`randLightServiceWithSquare` doesn't exist in this pkg anymore, same with `randFullServiceWithSquare`

Reply:
fixed to reference `light.RandServiceWithSquare` instead